qa/vagrant/src/test/resources/packaging/scripts/packaging_test_utils.bash

```diff
@@ -239,18 +239,62 @@ clean_before_test() {
 start_elasticsearch_service() {
     local desiredStatus=${1:-green}
 
+    run_elasticsearch_service 0
+
+    wait_for_elasticsearch_status $desiredStatus
+
+    if [ -r "/tmp/elasticsearch/elasticsearch.pid" ]; then
+        pid=$(cat /tmp/elasticsearch/elasticsearch.pid)
+        [ "x$pid" != "x" ] && [ "$pid" -gt 0 ]
+        echo "Looking for elasticsearch pid...."
+        ps $pid
+    elif is_systemd; then
+        run systemctl is-active elasticsearch.service
+        [ "$status" -eq 0 ]
+
+        run systemctl status elasticsearch.service
+        [ "$status" -eq 0 ]
+
+    elif is_sysvinit; then
+        run service elasticsearch status
+        [ "$status" -eq 0 ]
+    fi
+}
+
+# Start elasticsearch
+# $1 expected status code
+# $2 additional command line args
+run_elasticsearch_service() {
+    # Set the CONF_DIR setting in case we start as a service
```

Comment:
I'd probably save $1 and $2 to named variables because they can get confusing lower down.